packages/@aws-cdk/core/test/test.staging.ts

```diff
@@ -122,7 +122,7 @@ export = {
     const assembly = app.synth();
     test.deepEqual(
       readDockerStubInput(),
-      `run --rm ${USER_ARG} -v /input:/asset-input -v /output:/asset-output -w /asset-input alpine DOCKER_STUB_SUCCESS`,
+      `run --rm ${USER_ARG} -v /input:/asset-input:delegated -v /output:/asset-output:delegated -w /asset-input alpine DOCKER_STUB_SUCCESS`,
```

Comment:
Just to make sure, if this is specified in non macOS environments, it will be ignored?

Reply:
Yes

https://docs.docker.com/storage/bind-mounts/#choose-the--v-or---mount-flag

> This setting only applies to Docker Desktop for Mac, and is ignored on all other platforms.

Reply:
Thanks!